master-docker-nonstandard-2/master.cfg

```diff
@@ -635,25 +635,8 @@ f_big_test.addStep(
     )
 )
 # get the source tarball and extract it
-f_big_test.addStep(
-    steps.FileDownload(
-        mastersrc=util.Interpolate(
-            "/srv/buildbot/packages/"
-            + "%(prop:tarbuildnum)s"
-            + "/"
-            + "%(prop:mariadb_version)s"
-            + ".tar.gz"
-        ),
-        workerdest=util.Interpolate("%(prop:mariadb_version)s" + ".tar.gz"),
-    )
-)
-f_big_test.addStep(
-    steps.ShellCommand(
-        command=util.Interpolate(
-            "tar -xzf " + "%(prop:mariadb_version)s" + ".tar.gz --strip-components=1"
-        )
-    )
-)
+
```

Comment:
Can you remove this empty line? Also can you update the commit description to include the fact that the old variant fails on dev and all other builds use the utils function